src/services/notifyUserService.js

```diff
@@ -0,0 +1,49 @@
+import Model from '../models';
+
+const { Requests, Notifications } = Model;
+/**
+ *
+ *
+ * @class userService
+ */
+class notifyUserService {
+  /**
+   *
+   *
+   * @static
+   * @param {Object} newNotification
+   * @returns {Object} createdNotification
+   * @memberof notifyUserService
+   */
+  static async createNotification(newNotification) {
+    try {
+      const createdNotification = await Notifications.create(newNotification);
+      return createdNotification;
+    } catch (error) {
+      return false;
+    }
+  }
+
+  /**
+ *
+ *
+ * @static
+ * @param {Integer} id
+ * @param {String} updatePassword
+ * @returns {Object} User
+ * @memberof userService
+ */
+  static async getSpecificTrips(id) {
```

Comment:
I don't think this method is related to the NotificationService hence it shouldn't be in this class